cni-plugin/pkg/install/install_linux.go

```diff
@@ -30,7 +30,8 @@ func defaultNetConf() string {
       "mtu": __CNI_MTU__,
       "ipam": {"type": "calico-ipam"},
       "policy": {"type": "k8s"},
-      "kubernetes": {"kubeconfig": "__KUBECONFIG_FILEPATH__"}
+      "kubernetes": {"kubeconfig": "__KUBECONFIG_FILEPATH__"},
+      "require_mtu_file": __REQUIRE_MTU_FILE__
```

Comment:
we'll need to add this to the windows defaultNetConf as well: https://github.com/projectcalico/calico/blob/8ae53a944cdc35a3ff4311206bb8f729ecd881d1/cni-plugin/pkg/install/install_windows.go#L31

Reply:
upon looking at this again, it looks like the Windows dataplane does not write an MTU file at all: https://github.com/projectcalico/calico/blob/4254a471107a068363e48643cf5676ec5350f99c/felix/dataplane/windows/win_dataplane.go